services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchPortConfigSpec.groovy

```diff
@@ -65,34 +66,54 @@ class SwitchPortConfigSpec extends BaseSpecification {
             assert islUtils.getIslInfo(islUtils.reverseIsl(isl)).get().state == IslChangeType.DISCOVERED
         }
 
-        and: "Port Up event is logged in OTSDB"
+        and: "Port UP event is logged in OpenTSDB"
         Wrappers.wait(WAIT_OFFSET) {
             statsData = otsdb.query(portUpTime, "pen.switch.state",
                     [switchid: isl.srcSwitch.dpId.toOtsdFormat(), port: isl.srcPort]).dps
             assert statsData.size() == 1
         }
         statsData.values().first() == otsdbPortUp
+
+        where:
+        isl << uniqueIsls
```

Comment:
Do we need to do the same checks on destination switch, or the source switch should be enough?

Reply:
Source and destination switches here are used in a bit different context, since we are not working with 'flows'. For ISLs it actually doesn't matter src or dst switch is used, since there is always a reversed version of that ISL where src and dst are reversed. Or am I missing some idea?

Reply:
It is not important whether it is a source switch or not. Here we are collecting ISLs where sources switches will be unique switches depending on manufacturer and firmware version.  We can collect the same ISLs where such switches will be destination switches and bring ports DOWN/UP on them.

Reply:
> We can collect the same ISLs where such switches will be destination switches and bring ports DOWN/UP on them.

...but this makes no sense, at least for me

Reply:
Yes, I tried to say the same.